No longer works

`C:\GitHub> npm i -g generator-jrrreact
npm ERR! Windows_NT 10.0.14393
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\Logan\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "i" "-g" "generator-jrrreact"
npm ERR! node v7.2.0
npm ERR! npm  v3.10.5
npm ERR! code E404

npm ERR! 404 Registry returned 404 for GET on https://registry.npmjs.org/generator-jrrreact
npm ERR! 404
npm ERR! 404  'generator-jrrreact' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\GitHub\npm-debug.log`

Did you forget to push to npm?

--- a/README.md
+++ b/README.md
@@ -21,7 +21,7 @@ npm i -g yo
 ## Installation
 
 ```
-npm i -g generator-jrrreact
+npm i -g generator-jrreact
 ```
 
 ## Usage

